include/libtorrent/entry.hpp

```diff
@@ -95,8 +95,30 @@ namespace libtorrent
 			bool operator()(T1 const& rhs, T2 const& lhs) const
 			{ return rhs < lhs; }
 		};
-	};
+
+		template<class T>
+		class map_string : public std::map<std::string, T, aux::strview_less>
+		{
+		};
+#else
+		template<class T>
```

Comment:
original std::map::find string_view commit ( 48a647a1696e3029eed2fdd68fa9a691b9c79f10 ) assumes that it will work only at C++14